stashed/contributors.md

```diff
@@ -3,3 +3,4 @@ CodeforPDX:
 - Treasure Porth
 - Wade Christensen
 - Hugh Harker
+- Yahya Hafez
```

Comment:
The /stashed folder is actually the old version of the app! So let's add your name to the contributors.md at the root of the dwellingly-app folder